intro.adoc

```diff
@@ -5,17 +5,57 @@ Debugging and tracing are essential tools utilized by developers to identify and
     - *Debugging Restrictions:* The external debugger is restricted to halting the hart only when it is in a debug-allowed machine mode program or supervisor domain.
     - *Operations Constraints:* All debug operations, such as memory accesses and CSR accesses, are constrained by the per-hart control knobs.
     - *Memory Protection:* The external debugger cannot bypass the memory translation or protection mechanisms enforced on the hart.
+
+    - *System Bus Access Protection:* Optional. If implemented, System Bus Access must be protected by a bus initiator protection mechanism, such as IOPMP or WorldGuard.
+
+The sdtrig extension in the debug specification is extended to apply the enforcement:
+
     - *Trigger Constraints:* The firing or matching of triggers is constrained by the per-hart control knobs.
-    - *System Bus Access Protection (Optional):* If implemented, System Bus Access must be protected by a bus initiator protection mechanism, such as IOPMP or WorldGuard.
 
-The following enforcements are applied to the trace specifications:
+Regarding the trace specifications, these constraints are placed :
+
+    - *Per-Hart Trace Control Knobs (Smsdetrc extension cite:[smmtt]):* Dedicated per-hart control knobs are introduced for tracing, specifically for machine mode and supervisor domains.
 
-    - *Per-Hart Trace Control Knobs:* Dedicated per-hart control knobs are introduced for tracing, specifically for machine mode and supervisor domains.
     - *Trace Encoder Control:* The trace encoder is enabled or disabled by a sideband signal from the hart to the trace encoder, controlled according to the per-hart knobs.
 
+=== Terminology
+
+*Abstract command*: a high-level command in Debug Module used to interact with and control harts
+
+*CSR*: control/status register
+
+*Debug Access Privilege*:  the privilege with which abstract commands or instructions in program buffers access hardware resources 
+
+*Debug Mode*: an additional privilege mode to support off-chip debugging
+
+*Debug Module*: a RISC-V module translates abstract debug operations and performs them to the associated harts
+
+*External Debug*: debug software running in a host CPU, debugging the RISC-V platform via a debug transport like JTAG
+
+*Hart*: a RISC-V hardware thread
+
+*IOPMP*: Input-Output Physical Memory Protection
+
+*ISA*: instruction set architecture
+
+*Machine mode*: the most privileged mode in the RISC-V privilege model
+
+*PMA*: properties and capabilities of each region of the  physical address space
+
+*PMP*: physical memory protection unit provides per-hart machine-mode control for each physical memory region
+
+*Privilege level*: provides protection between different components of the RISC-V software stack
+
+*Program buffer*: a buffer in Debug Module to execute arbitrary instructions on a hart
+
+*Supervisor domain*: a privileged execution environment isolated by Supervisor Domain Access Protection
```

Comment:
This is a RISC-V privileged architecture
extension, define in Smmtt specification, to support
physical address space (memory and devices)
isolation for more than one supervisor domain.
Supervisor domains enable the reduction of the
supervisor Trusted Computing Base (TCB), with
differentiated access to memory and other
platform resources